src/Stryker.CLI/Stryker.CLI/FileBasedInput.cs

```diff
@@ -86,6 +86,9 @@ public class FileBasedInput : IExtraData
         [JsonPropertyName("ignore-methods")]
         public string[] IgnoreMethods { get; init; }
 
+        [JsonPropertyName("output-html-report-name")]
```

Comment:
I would prefer something like 'report-file-name' as the config property. We could layer also use this to change the file name of the json report.

Reply:
It should be updated as such now!